doc/apps/panel.md

```diff
@@ -0,0 +1,29 @@
+# Panel
+
+Ploomber Cloud supports [Panel](https://github.com/holoviz/panel). For information on how to develop Panel apps, [please check the documentation](https://panel.holoviz.org/).
+
+To deploy a Panel app you need at least two files:
+
+1. Your application file (`app.py`)
+2. A dependencies file (`requirements.txt`)
+
+## Application file
+
+Your `app.py` must be a Panel application. An example is available [here.](https://github.com/ploomber/doc/blob/main/examples/panel/data-viz/app.py)
+
+## Dependencies
+
+To deploy a new project, list your dependencies in a (`requirements.txt`). For example, if you're using [JupySQL](https://jupysql.ploomber.io), pandas and matplotlib, your `requirements.txt` file will look like this:
+
+```
+# sample requirements.txt
+jupysql
+pandas
+matplotlib
+```
+
```

Comment:
Since we already host the example app.zip, let's just put it so user can download immediately and playaround
```suggestion
## Complate Example zip
You can download our complete example [app.zip](https://github.com/ploomber/doc/blob/main/examples/panel/data-viz/app.zip) file to play with
```